docs/guides/src/main/server/configuration_overview.adoc

```diff
@@ -0,0 +1,173 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Configuring Keycloak"
+summary="An overview of the server configuration">
+
+In this guide, you will learn the core concepts around the server configuration and how to configure the server using the different
+configuration options available. You will also learn how to properly configure the server to achieve an optimal runtime for faster
+startup and low memory footprint.
+
+== Understanding Server Options
+
+All the available configuration options can be found by either running the `help` command for individual commands:
```

Comment:
Eliminate "the available" which adds no meaning.
Eliminate "either" because of the suggested change on the next line.